index.html

```diff
@@ -39,94 +43,111 @@ <h1 class="greet">Hey There. <br> I’m Alex</h1>
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-
-    </div>
-
-    <div class="professional-art art2">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-    </div>
-
-    <div class="professional-art art3">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-    </div>
-
-    <div class="professional-art art4">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-    </div>
-
-    <div class="professional-art art5">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
+      <div class="line"></div>
     </div>
+    <div class="work-container">
+      <div class="multi-post">
+        <img src="icons/images/Img Placeholder1.png" alt="illustration image">
+        <div>
+          <h2 id="multi-head-two">Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul>
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="multi-btn">See Project</button>
```

Comment:
- [ ] Kindly use the right button interaction style for all the buttons in the page as showin in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213470035-295109ed-42cb-4826-a1b8-70d57c069117.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213470010-7d77a191-c2ea-4c6d-a36f-719394f72d39.png)